projects/js-packages/licensing/components/activation-screen-controls/style.scss

```diff
@@ -39,11 +39,12 @@
 	.jp-license-activation-screen-controls--license-field,
 	.jp-license-activation-screen-controls--license-field-with-error {
 		label {
-			font-weight: 600;
-			font-size: var(--font-body);
+			font-weight: 600 !important;
```

Comment:
I have trouble overriding the label's font styling in the Select component. the `!important` simplifies the fix. Though not desirable but this should be safe solution.

Reply:
I _highly_ recommend against using `!important`, and if I'm not mistaken, it's all but forbidden in our coding conventions. Instead of this approach, add extra specifiers to the rule until it "sticks." For example, we can do multiple classes, or use more parent or child elements, or even add an ID if it's feasible.

Reply:
@jkguidaven After some experimentation, I found that using `.components-input-control__label` instead of `label` gives the necessary specificity to override the rule causing the issue. That is:

```css
  .jp-license-activation-screen-controls--license-field,
  .jp-license-activation-screen-controls--license-field-with-error {
    .components-input-control__label {
```

Reply:
@mattgawarecki I actually used `.components-input-control__label` before and give up. For some reason the styling doesn't get applied due to some arbitrary class name.. 🤔 I am not sure what I am missing. I used this `jetpack build packages/my-jetpack` command to build it. 

<img width="455" alt="Screen Shot 2022-11-29 at 2 34 05 PM" src="https://user-images.githubusercontent.com/56598660/204456883-7b506375-7ca2-4b82-a902-e255805b4947.png">


This arbitrary class name (.css-1imalal) is added to the label which overrides my styling. I can't figure out how I can prevent this from happening aside using `!important`.

<img width="482" alt="Screen Shot 2022-11-29 at 2 39 39 PM" src="https://user-images.githubusercontent.com/56598660/204457276-3495c5f9-43e9-4ddd-974e-37b8756bb5e8.png">

Reply:
I went back to test again and I see what you mean. Another alternative solution here is to apply the same class name multiple times (which is what's happening inside the WordPress label component to apply `css-1imalal` three times). For example:

```sass
.jp-license-activation-screen-controls--license-field,
.jp-license-activation-screen-controls--license-field-with-error {
    .components-input-control__label.components-input-control__label.components-input-control__label {
```

I'm not sure this is really an improvement over `!important`, but it guards against future usages of `!important`, which can really make styling code hard to work with in a codebase as large as Jetpack. I leave the final decision up to you. 🙂

Reply:
thanks @mattgawarecki, I didn't know this approach and it work 👍